lib/model/query/blobs.js

```diff
@@ -120,14 +120,14 @@ const _purgeUnattachedNotUploaded = () => ({ all }) => all(sql`
       AND blobs.id = b.id
 `);
 
-const _purgeOneUnattachedUploaded = () => ({ maybeOne }) => maybeOne(sql`
+const _purgeUnattachedUploadedBatch = () => ({ all }) => all(sql`
```

Comment:
Is this batch purging supposed to only be for this part where the blobs are uploaded to S3? There was some earlier version of this PR where the unattached clause was modified and it affected all the blob purging so I'm not sure if batching those other queries should be part of this PR or not.

Reply:
As per PR title, this is just batching the delete of uploaded blobs.